app/_data/tables/install_options_34x.yml

```diff
@@ -1,84 +1,89 @@
 columns:
-  - name: Kong OSS
+  - name: Platform
+  - name: Open Source
+    subtitle: An open-source package containing the basic API <br />gateway functionality and open-source plugins.
     key: oss
   - name: Kong Gateway Enterprise
+    subtitle: Kong’s API gateway with added functionality.
     key: enterprise
-  - name: Officially Supported
-    key: support
 features:
-  - name: "Packages"
+  - name: "Docker"
     items:
-      - name: "Debian"
-        url: /gateway/VERSION/install/linux/debian/
+      - name: "Debian (bullseye-slim)"
         oss: true
         enterprise: true
         support: true
+        url: /gateway/VERSION/install/docker/build-custom-images/
```

Comment:
Why is this a link to building custom images rather than a dockerhub search like amazonlinux is? Same question for RHEL

Reply:
same for ubuntu right?